html/pdf.php

```diff
@@ -48,7 +48,7 @@
 
 if (isset($_GET['report']) && !empty($_GET['report'])) {
     $report = mres($_GET['report']);
-    $pdf->SetHeaderData('../../'.$config['title_image'], 40, ucfirst($report), $config['project_name'], array(0, 0, 0), array(0, 64, 128));
+    $pdf->SetHeaderData('../../../../../html/'.$config['title_image'], 40, ucfirst($report), $config['project_name'], array(0, 0, 0), array(0, 64, 128));
```

Comment:
Because the pdf library already has it's path set so it ends up being /opt/librenms/vendor/phppdf/dasa/dasdas/dasdas/opt/librenms/html/